recipes/hdf5/all/test_package/CMakeLists.txt

```diff
@@ -11,6 +11,9 @@ endif()
 find_package(HDF5 COMPONENTS ${HDF5_COMPONENTS})
 
 add_executable(${PROJECT_NAME} test_package.c)
+if(HDF5_CXX)
+    target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

Comment:
Shouldn't this be handled by the toolchain? or are we still not requiring `cppstd` to be set

Reply:
compiler.cppstd is not mandatory in profile, and a good CMakeLists must express usage requirements.